src/config.json

```diff
@@ -0,0 +1,70 @@
+{
+    "initialMapCenter": [39.944558, -75.162605],
+    "initialMapZoom": 3,
+    "mapTilesMinZoom": 0,
+    "mapTilesMaxZoom": 18,
+    "mapTilesUrl": "https://server.arcgisonline.com/ArcGIS/rest/services/World_Topo_Map/MapServer/tile/{z}/{y}/{x}",
+    "mapTilesAttribution": "Tiles &copy; Esri &mdash; Esri, DeLorme, NAVTEQ, TomTom, Intermap, iPC, USGS, FAO, NPS, NRCAN, GeoBase, Kadaster NL, Ordnance Survey, Esri Japan, METI, Esri China (Hong Kong), and the GIS User Community",
```

Comment:
Consider using `basemap` as the prefix for these variables instead of `mapTiles`.